src/odb/src/def/BUILD

```diff
@@ -54,7 +54,7 @@ cc_library(
     ],
     hdrs = DEF_HEADERS,
     # Ignore warnings in generated code
-    copts = ["-Wno-unused-but-set-variable"],
+    copts = ["-Wno-unused-but-set-variable", "-Wno-modules-import-nested-redundant"],
```

Comment:
what is this warning?

Reply:
Unclear. Something about bcr bison hardcoding YYDEBUG to 1 and this causing problems with include if stdio.h and C++ modules due to some namespace scoping or something...